widget/tabcontainer.go

```diff
@@ -163,6 +163,25 @@ func (t *TabContainer) buildTabBar(buttons []fyne.CanvasObject) *fyne.Container
 	return tabBar
 }
 
+
+// Append adds a new CanvasObject to the end of the group
+func (t *TabContainer) Append(item *TabItem) {
+	r := Renderer(t).(*tabContainerRenderer)
+	t.Items = append(t.Items, item)
+	r.tabBar.Objects = append(r.tabBar.Objects, t.makeButton(item))
+
+	t.CreateRenderer().Refresh()
+}
+
+// Remove tab by index
+func (t *TabContainer) Remove(index int) {
```

Comment:
To be consistent with the methods above it should be RemoveIndex if you are passing an int.
Remove() would take a *TabItem.

We are trying to ensure consistency to all public API, sorry for the nit-picking

Reply:
Done:)